packages/dxf-deserializer/index.js

```diff
@@ -0,0 +1,592 @@
+/*
+## License
+
+Copyright (c) 2017 Z3 Development https://github.com/z3dev
+
+All code released under MIT license
+
+*/
+
+const {colorIndex, BYLAYER, getTLA} = require('./autocad')
+const dxf = require('./DxfReader')
+const {instantiateAsciiDxf} = require('./instantiate')
+const translateAsciiDxf = require('./translate')
+
+// //////////////////////////////////////////
+//
+// DXF (Drawing Exchange Format) is a CAD data file format developed by Autodesk
+//
+// //////////////////////////////////////////
+
+function handleError (e) {
+  console.log('error: line ' + e.line + ', column ' + e.column + ', bad character [' + e.c + ']')
```

Comment:
Errors should throw like for the other de/serializers , otherwise there is no way to handle the errors in the rest of the app or propagate them.
Same note as before for console.log output

Reply:
Changes made.
All logging of debug statements are commented out.
Error handling revised.